mitmproxy/flowfilter.py

```diff
@@ -492,25 +493,42 @@ def _make():
 def parse(s):
     # type: (str) -> TFilter
     try:
-        filt = bnf.parseString(s, parseAll=True)[0]
-        filt.pattern = s
-        return filt
+        filter = bnf.parseString(s, parseAll=True)[0]
+        filter.pattern = s
+        return filter
     except pp.ParseException:
         return None
     except ValueError:
         return None
 
 
+def match(flow, flt):
```

Comment:
I would prefer a `match(flt, flow):` signature, as that would be in line with `re.match`. Thoughts?